terraform/environments/delius-core/modules/environment_all_components/component.tf

```diff
@@ -18,3 +18,153 @@ resource "aws_security_group" "sg_db" {
   tags        = {}
 }
 
+resource "aws_security_group" "base_ami_test_instance_sg" {
```

Comment:
If all these resources relate the delius db,  do they belong in delius_core_db.tf?